core/kazoo_stdlib/src/kz_module.erl

```diff
@@ -0,0 +1,47 @@
+%%%-----------------------------------------------------------------------------
+%%% @copyright (C) 2018-, 2600Hz
+%%% @doc
+%%% @end
+%%%-----------------------------------------------------------------------------
+-module(kz_module).
+
+-export([is_exported/3
+        ,ensure_loaded/1
+        ]).
+
+%% @doc lifted from erlang ML
+%% http://erlang.2086793.n4.nabble.com/What-is-the-fastest-way-to-check-if-a-function-is-defined-in-a-module-tp4723096p4723108.html
+-spec is_exported(module(), atom(), arity()) -> boolean().
+is_exported(Module, Function, Arity)
+  when is_atom(Module),
+       is_atom(Function),
+       is_integer(Arity),
+       Arity >= 0 ->
```

Comment:
maybe do `Module` convert here and accept `ne_binary`.
` Module = kz_term:to_atom(Mod, 'true'),`

Reply:
I would rather this be strict since it is new functionality to use.